.github/workflows/docs.yml

```diff
@@ -13,8 +13,14 @@ jobs:
       - name: Checkout the code
         uses: actions/checkout@v2
 
-      - name: Build the docs
-        run: cargo doc --lib --all-features
+      - name: Build the docs (nightly)
+        run: |
+          rustup install nightly
+          cargo +nightly doc --lib --all-features
+
+      - name: Build the docs (stable)
+        run: cargo +stable doc --lib --all-features
```

Comment:
why are we building both nightly and stable docs?

Reply:
I described this in the dpr description, the stable docs will only fire if nightly fails, as I know we've had issues in the past with nightly not working correctly.

Reply:
Are you sure that's how the script is configured? It seems to me that `run: cargo +stable` goes before `if`, where it should be the other way around?

Reply:
Yay declarative yaml! The order of the arguments doesn't actually matter in this case. https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-status-check-functions. This is the reference I used to make myself believe it worked in the first place.